### PR TITLE
C#: Minor updates to Hot Reload documentation

### PR DESCRIPTION
**Pull request by isadorasophia**
Now that the documentation has been updated with Hot Reload information, I added some small corrections around grammar and consistency that I originally missed.

### CODE REVIEW

File: docs/csharp/debugging.md [hunk 1]
```diff
@@ -200,17 +200,17 @@ With the C# Dev Kit extension installed, the debugger allows you to apply C# cod
 
 ![Hot Reload displayed in the debugging toolbar](images/debugging/hotreload-toolbar.png)
 
-In order to enable Hot Reload, `csharp.experimental.debug.hotReload` must be set to true, see [User Settings](#user-settings) for more information. The Hot Reload session will only start if the target debugger engine supports applying code changes.
+In order to enable Hot Reload, `csharp.experimental.debug.hotReload` must be set to true, see [user settings](#user-settings) for more information. The Hot Reload session will only start if the target debugger engine supports applying code changes.
 
-### Supported .NET apps and scenarios
+### Supported projects and scenarios
```
Comment (gregvanl) [inline docs/csharp/debugging.md:205]: When you change an H2 or H3 header, you should check for direct links. In this case, search for "supported-net-apps-and-scenarios" and update to "supported-projects-and-scenarios"
I see at least one link in the FAQ to this header

File: docs/csharp/debugging.md [hunk 2]
```diff
@@ -200,17 +200,17 @@ With the C# Dev Kit extension installed, the debugger allows you to apply C# cod
 
 ![Hot Reload displayed in the debugging toolbar](images/debugging/hotreload-toolbar.png)
 
-In order to enable Hot Reload, `csharp.experimental.debug.hotReload` must be set to true, see [User Settings](#user-settings) for more information. The Hot Reload session will only start if the target debugger engine supports applying code changes.
+In order to enable Hot Reload, `csharp.experimental.debug.hotReload` must be set to true, see [user settings](#user-settings) for more information. The Hot Reload session will only start if the target debugger engine supports applying code changes.
```
Comment (gregvanl) [inline docs/csharp/debugging.md:203]: The header is "User settings" so you can use that casing in the link

**Comment by gregvanl** [comment]
Looks good. Just a couple of comments about matching header casing and searching for header links that need to be updated

File: docs/csharp/debugging.md (hunk 2 above)
Comment (isadorasophia) [reply docs/csharp/debugging.md:203]: got it, I'll correct the other references to that then

**Review by gregvanl — APPROVED** [review]
Thanks!

**Pull request merged**